dataland-frontend/tests/e2e/specs/eu-taxonomy-non-financials/DataUpload.ts

```diff
@@ -23,15 +21,18 @@ describeIf(
   },
   function () {
     let frontendDocumentHash = "";
-    let testData: FixtureData<EuTaxonomyDataForNonFinancials>;
     const uploadReports = new UploadReports("referencedReports");
-    before(function () {
-      cy.fixture("CompanyInformationWithEuTaxonomyDataForNonFinancials").then(function (
-        jsonContent: FixtureData<EuTaxonomyDataForNonFinancials>[],
-      ) {
-        testData = jsonContent[0];
-      });
-    });
+
+    /**
+     * Fills all the required fields of the eu-taxonomy upload form for non-financial companies to enable submit button
+     */
+    function fillRequiredEutaxonomyNonFinancialsFields(): void {
+      cy.get('[data-test="fiscalYearEnd"] button').should("have.class", "p-datepicker-trigger").click();
+      cy.get("div.p-datepicker").find('button[aria-label="Next Month"]').click();
+      cy.get("div.p-datepicker").find('span:contains("11")').click();
+      cy.get('select[name="value"]').select(1);
```

Comment:
Is this for selecting the report regarding assurance. Shouldn't we be a bit more explicit with the selector?

Reply:
Ja genau dafür ist es.

Ja ich fand das auch ziemlich ugly und unintuitiv, wollte jetzt aber nicht noch mehr scope creep machen.

Reply:
Alles klar. Ist ja Copy Paste.